salesforce/models.py

```diff
@@ -386,7 +386,6 @@ class ResourceDownload(models.Model):
         ('Bookshare', 'Bookshare'),
         ('Chegg Reader', 'Chegg Reader'),
     )
-    salesforce_id = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
likewise, can probably remove the number of times accessed and the `save()` method override?

Reply:
Good catch. it is removed.